Confus/Confus/Health.h

```diff
@@ -10,6 +10,7 @@ namespace Confus
 		std::function<bool(irr::scene::ISceneNode* a_DamagedNode)> m_DeathCallback;
 		irr::scene::ISceneNode* m_AttachedNode;
 		int m_Health;
+		int m_MaxHealth = 100;
```

Comment:
We can already change the health by using heal and damage, so I don't see the need for making it public. Can do for MaxHealth, but there's less harm in keeping it encapsulated like this. If needed in the future, you can always add a setter